core/src/main/java/org/apache/cxf/common/util/PackageUtils.java

```diff
@@ -85,13 +86,8 @@ public static String getSharedPackageName(List<Class<?>> classes) {
     }
 
     public static String parsePackageName(String namespace, String defaultPackageName) {
-        String packageName = (defaultPackageName != null && defaultPackageName.trim().length() > 0)
-            ? defaultPackageName : null;
-
-        if (packageName == null) {
-            packageName = getPackageNameByNameSpaceURI(namespace);
-        }
-        return packageName;
+        return (defaultPackageName != null && !defaultPackageName.isEmpty())
```

Comment:
Not sure how important it is in this context, but `.trim()` is lost: `!defaultPackageName.trim().isEmpty()`. `" ".isEmpty == false`, whereas  `" ".trim().isEmpty == true`